ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,93 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: platform.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: platform-cluster-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
+  nuxeo:
+    customParams: |-
+      elasticsearch.restClient.socketTimeoutMs=300000
+      elasticsearch.restClient.connectionTimeoutMs=300000
+    podAnnotations:
+      preview/scm.ref: ${SCM_REF}
+    podLabels:
+      branch: "$BRANCH_NAME"
+      team: platform
+      resource: pod
+      usage: preview
+    service:
+      annotations:
+        fabric8.io/ingress.annotations: |-
+          nginx.ingress.kubernetes.io/proxy-body-size: 0
+          nginx.ingress.kubernetes.io/server-snippet: ignore_invalid_headers off;
+      labels:
+        team: platform
+        resource: service
+        usage: preview
+    image:
+      repository: ${DOCKER_REGISTRY}/nuxeo/nuxeo
+      tag: ${VERSION}
+      pullPolicy: Always
+    packages: nuxeo-web-ui nuxeo-dam
+    livenessProbe:
+      initialDelaySeconds: 900
+      periodSeconds: 20
+      successThreshold: 1
+      failureThreshold: 5
+      timeoutSeconds: 10
+    readinessProbe:
+      periodSeconds: 20
+      initialDelaySeconds: 600
+      successThreshold: 1
+      timeoutSeconds: 10
+      failureThreshold: 5
```

Comment:
This may be linked to my other comment about `preview-health-timeout`, these values are for an environment deploying nuxeo 10.10. On master we probably don't need to wait 10min before checking readiness.

I think we could try the default values coming from nuxeo helm chart.

Reply:
I lowered --preview-health-timeout to 5m
Seems to be working fine.

Then I realized this is the default value so I'll remove the whole jx preview option on next push